tests/cephadm/test_cephadm_pull.py

```diff
@@ -0,0 +1,70 @@
+import ast
+
+import requests
+import yaml
+
+from ceph.ceph_admin import CephAdmin
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def getting_ceph_version(link):
+    """
+    Method return ceph version from magna002 latest ceph container yaml
+    Args:
+        link: magna002 latest ceph container yaml link
+    Returns:
+        ceph version
+    """
+
+    response = requests.get(link)
+    ceph_versions = yaml.safe_load(response.text)
+    ceph_version_5 = []
+    for ceph_version in ceph_versions:
+        if ceph_version >= 5:
```

Comment:
In the logic consider only last 5.x release means 5.1, 5.2 then cephadm pull shows 5.2 ceph version.
like that code it.
also attach test result logs for the changes.

Reply:
Thanks @sunilangadi2 
As discussed updated logic to take latest version.

Log : http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-2EHTYK/